ptsprojects/zephyr/sm.py

```diff
@@ -239,7 +239,6 @@ def test_cases(pts):
                   [TestFunc(btp.gap_set_io_cap, IOCap.display_only)],
                   generic_wid_hdl=sm_wid_hdl),
         ZTestCase("SM", "SM/MAS/OOB/BV-01-C",
-                  # Set CONFIG_BT_SMP_OOB_LEGACY_PAIR_ONLY=y in prj.conf
```

Comment:
Kardes, check with:
https://github.com/AyturkDuzen/zephyr/pull/2

It seems that we do not need the kconfig anymore

Reply:
yes I also tried with both softdevice controller and software based. No need for extra config